java/bulk-export/bulk-export-task-execution/src/main/java/sleeper/bulkexport/taskexecution/ECSBulkExportTaskRunner.java

```diff
@@ -105,29 +105,41 @@ public static void main(String[] args) throws ObjectFactoryException, IOExceptio
      * This method initialises the SQS bulk export queue handler and processes
      * messages from the SQS queue for bulk export tasks.
      *
-     * @param  instanceProperties        the instance properties
-     * @param  tablePropertiesProvider   a table properties provider
-     * @param  sqsClient                 an SQS client
-     * @param  s3Client                  an S3 client
-     * @param  dynamoDBClient            a DynamoDB client
-     * @param  hadoopConf                a Hadoop configuration
-     * @throws IOException               if there is an error interacting with S3
-     * @throws IteratorCreationException if there is an error creating iterators
-     * @throws ObjectFactoryException    if there is an error creating objects
+     * @param  instanceProperties      the instance properties
+     * @param  tablePropertiesProvider a table properties provider
+     * @param  sqsClient               an SQS client
+     * @param  s3Client                an S3 client
+     * @param  dynamoDBClient          a DynamoDB client
+     * @param  hadoopConf              a Hadoop configuration
+     * @throws IOException             if there is an error interacting with S3
+     * @throws IllegalStateException   if there is an error receiving the message and deseralising it
      */
     public static void runECSBulkExportTaskRunner(
             InstanceProperties instanceProperties, TablePropertiesProvider tablePropertiesProvider,
-            SqsClient sqsClient, S3Client s3Client, DynamoDbClient dynamoDBClient, Configuration hadoopConf) throws IOException, IteratorCreationException, ObjectFactoryException {
+            SqsClient sqsClient, S3Client s3Client, DynamoDbClient dynamoDBClient,
+            Configuration hadoopConf) throws IOException, IllegalStateException {
         SqsBulkExportQueueHandler exportQueueHandler = new SqsBulkExportQueueHandler(sqsClient,
                 tablePropertiesProvider, instanceProperties);
         LOGGER.info("Waiting for leaf partition bulk export job from queue {}",
                 instanceProperties.get(CdkDefinedInstanceProperty.LEAF_PARTITION_BULK_EXPORT_QUEUE_URL));
+        Optional<SqsMessageHandle> messageHandleOpt;
+
+        try {
+            messageHandleOpt = exportQueueHandler.receiveMessage();
+        } catch (IOException | IllegalStateException e) {
+            LOGGER.error("There is a problem getting the message.", e);
+            throw e;
```

Comment:
It doesn't look like it's possible for this to ever throw an IllegalStateException. It doesn't look like that exception is relevant here.

Is there a benefit to catching and rethrowing here in general? I think the stack trace will still be logged if it's thrown all the way out.